packages/superset-ui-time-format/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superset-ui/time-format",
-  "version": "0.11.0",
+  "version": "0.11.1",
```

Comment:
The version number is managed automatically by `lerna`. Please do not modify the version number manually.

Reply:
@kristw I'm reverting this line.  Do I need to do anything to increment the version number at all?  Thanks.

Reply:
I can `yarn release` after the change is merged to up the version number, Thomas.

Reply:
@xtinec sounds good